arch/platform/nrf/Makefile.nrf

```diff
@@ -0,0 +1,20 @@
+ifndef CONTIKI
+  $(error CONTIKI not defined! You must specify where CONTIKI resides!)
+endif
+
+### Board and BSP selection
+BOARD ?= nrf53/pdk/application
+BOARDS = nrf53/pdk/application nrf53/pdk/network nrf52/pdk
```

Comment:
You forgot to update the board names here

Reply:
Thank you.